crates/goose-mcp/Cargo.toml

```diff
@@ -27,7 +27,7 @@ schemars = "1.0"
 lazy_static = "1.5"
 shellexpand = "3.1.0"
 indoc = "2.0.5"
-xcap = "0.0.14"
+xcap = "=0.4.0"
```

Comment:
this is the revlock version that allows everything else to pass. Movement past this in #6301